yucca/evaluation/YuccaEvaluator.py

```diff
@@ -130,7 +142,7 @@ def __init__(
             self.labels = ["0", "1"]
             self.name += "_BINARY"
 
-        self.labelarr = np.array(self.labels, dtype=np.uint8)
+        self.labelarr = np.sort(np.array(self.labels, dtype=np.uint8))
```

Comment:
Personally i am not a fan of specifying the type in the variable name. If it is really important, i would use a type hint:

```python
s: str = ''
```